src/domexpression.js

```diff
@@ -159,6 +159,39 @@ var _ = Mavo.DOMExpression = $.Class({
 					return "";
 				}
 
+				if (env.value instanceof Promise) {
+					env.value
+						.then(value => {
+							env.value = Mavo.Primitive.format(value, {
+								attribute: this.attribute,
+								element: this.element
+							});
+
+							this.output(env.value);
+
+							Mavo.hooks.run("domexpression-update-end", env);
```

Comment:
Should we run the hook here? I think yes, that's why I added it. But what about the `domexpression-update-start ` hook? Should we run it before changing `env.value` when the promise is resolved or rejected?